<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>SCDNEY - Publications</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./images/USydLogo.svg" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">SCDNEY</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html" rel="" target="">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html" rel="" target="">
 <span class="menu-text">About us</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./software.html" rel="" target="">
 <span class="menu-text">Software</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./Publications.html" rel="" target="" aria-current="page">
 <span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./workshops.html" rel="" target="">
 <span class="menu-text">Problem-focused Workshops</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#methodology-and-tools" id="toc-methodology-and-tools" class="nav-link active" data-scroll-target="#methodology-and-tools"><strong>Methodology and tools</strong></a>
  <ul class="collapse">
  <li><a href="#preprint" id="toc-preprint" class="nav-link" data-scroll-target="#preprint">Preprint</a></li>
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section">2023</a></li>
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1">2022</a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2">2021</a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3">2020</a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4">2019</a></li>
  <li><a href="#section-5" id="toc-section-5" class="nav-link" data-scroll-target="#section-5">2018</a></li>
  <li><a href="#section-6" id="toc-section-6" class="nav-link" data-scroll-target="#section-6">2016</a></li>
  </ul></li>
  <li><a href="#applications" id="toc-applications" class="nav-link" data-scroll-target="#applications"><strong>Applications</strong></a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Publications</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="methodology-and-tools" class="level2">
<h2 class="anchored" data-anchor-id="methodology-and-tools"><strong>Methodology and tools</strong></h2>
<section id="preprint" class="level3">
<h3 class="anchored" data-anchor-id="preprint">Preprint</h3>
<ul>
<li><p><font size="+1"> <strong>Lin, Y.,</strong> Wu, T. Y., Chen, X., Wan, S., Chao, B., Xin, J., <strong>Yang, J.Y.H &amp; Wang,</strong> Wong W.H., Wang R.Y.X. (2022). scTIE: data integration and inference of gene regulation using single-cell temporal multimodal data. <em>BioRxiv</em>. [<a href="https://www.biorxiv.org/content/10.1101/2023.05.18.541381v1">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yin, D.</strong>, <strong>Cao, Y.</strong>, <strong>Chen, J.</strong>, Mak, C. L., Yu, K. H., Lin, Y., <strong>Ho, J.W.K.</strong> &amp; <strong>Yang, J.Y.H.</strong> (2022). Covidscope: An atlas-scale COVID-19 resource for single-cell meta analysis at sample and cell levels. <em>BioRxiv</em>. [<a href="https://doi.org/10.1101/2022.12.03.518997">paper</a>]</font></p></li>
</ul>
</section>
<section id="section" class="level3">
<h3 class="anchored" data-anchor-id="section">2023</h3>
<ul>
<li><p><font size="+1"> <strong>Couto, B. Z. P.,</strong> <strong>Robertson, N.,</strong> <strong>Patrick, E.,</strong> &amp; <strong>Ghazanfar, S.</strong> (2023). MoleculeExperiment enables consistent infrastructure for molecule-resolved spatial transcriptomics data in Bioconductor. <em>Accepted by Bioinformatics</em> [<a href="https://www.biorxiv.org/content/10.1101/2023.05.16.541040v1">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yu, L.,</strong> <strong>Liu, C.,</strong> <strong>Yang, J.Y.H.,</strong> &amp; <strong>Yang, P.</strong> (2023). Ensemble deep learning of embeddings for clustering multimodal single-cell omics data. <em>Bioinformatics</em>. [<a href="https://academic.oup.com/bioinformatics/article/39/6/btad382/7197799">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y.</strong>, Cao, Y., Willie, E., Patrick, E., &amp; Yang, J. Y. H. (2023). Atlas-scale single-cell multi-sample multi-condition data integration using scMerge2. <em>Nature Communications</em>. [<a href="https://www.nature.com/articles/s41467-023-39923-2">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Ghazanfar, S.,</strong> Guibentif, C., &amp; Marioni, J. C. (2023). Stabilized mosaic single-cell data integration using unshared features. Nature Biotechnology, 1-9. [<a href="https://www.nature.com/articles/s41587-023-01766-z">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Liu, C.,</strong> <strong>Huang, H.,</strong> &amp; <strong>Yang, P.</strong> (2023). Multi-task learning from multimodal single-cell omics with Matilda. <em>Nucleic Acids Research</em>, 51(8), e45-e45. [<a href="https://academic.oup.com/nar/article/51/8/e45/7076464">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Patrick, E., Canete, N. P., Baharlou, H., Iyengar, S. S.,</strong> Harman, A. N., Sutherland, G. T., and <strong>Yang P.</strong> (2021). Spatial analysis for highly multiplexed imaging data to identify tissue microenvironments. <em>Cytometry A</em>. [<a href="http://bioconductor.org/packages/release/bioc/html/lisaClust.html">package</a>][<a href="http://shiny.maths.usyd.edu.au/lisaClust">shiny</a>][<a href="https://doi.org/10.1101/2021.08.16.456469">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y.</strong>, Ghazanfar, S., Yang, P., &amp; <strong>Yang, J.Y.H.</strong> (2023). Benchmarking of analytical combinations for COVID-19 outcome prediction using single-cell RNA sequencing data. <em>Briefings in Bioinformatics</em>. [<a href="https://doi.org/10.1093/bib/bbad159">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y.</strong>, <strong>Tran, A.</strong>, <strong>Kim, H.</strong>, Robertson, N., Lin, Y., Torkel, M., Yang, P., Patrick, E., <strong>Ghazanfar, S.</strong>, &amp; <strong>Yang, J.Y.H.</strong> (2023). Thinking process templates for constructing data stories with SCDNEY. <em>F1000 Research</em>. [<a href="https://doi.org/10.12688/f1000research.130623.1">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yu, L.</strong>, Cao, Y., Yang, J.Y.H., &amp; <strong>Yang, P</strong>. (2022). Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data. <em>Genome Biology</em>. [<a href="https://doi.org/10.1186/s13059-022-02622-0">paper</a>]</font></p></li>
</ul>
</section>
<section id="section-1" class="level3">
<h3 class="anchored" data-anchor-id="section-1">2022</h3>
<ul>
<li><p><font size="+1"> <strong>Canete, N. P., Iyengar, S. S.</strong>, Wilmott. J. S., <strong>Ormerod, J. T.</strong>, Harman, A. N., and <strong>Patrick, E.</strong>(2021) spicyR: Spatial analysis of in situ cytometry data in R. <em>Bioinformatics</em>. [<a href="http://bioconductor.org/packages/release/bioc/html/spicyR.html">package</a>][<a href="http://shiny.maths.usyd.edu.au/spicyR/">shiny</a>][<a href="https://doi.org/10.1093/bioinformatics/btac268">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y., Lin, Y., Patrick, E., Yang, P., and Yang, J.Y.H.</strong> (2022) scFeatures: Multi-view representations of single-cell and spatial data for disease outcome prediction. <em>Bioinformatics</em>. [<a href="https://doi.org/10.1093/bioinformatics/btac590">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y.</strong>, Wu, T.Y., Wan, S., <strong>Yang, J.Y.H.,</strong> Wong, W.H., and <strong>Wang, Y.X.R.</strong> (2022) scJoint: transfer learning for data integration of single-cell RNA-seq and ATAC-seq. <em>Nature Biotechnology</em>. [<a href="https://github.com/SydneyBioX/scJoint">package</a>][<a href="https://doi.org/10.1038/s41587-021-01161-6">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Tran, A., Yang P., Yang, J.Y.H., and Ormerod, J. T.</strong> (2022) scREMOTE: Using multimodal single cell data to predict regulatory gene relationships and to build a computational cell reprogramming model. <em>NAR Genomics and Bioinformaitics</em>. [<a href="https://github.com/SydneyBioX/scREMOTE">package</a>][<a href="https://doi.org/10.1093/nargab/lqac023">paper</a>]</font></p></li>
</ul>
</section>
<section id="section-2" class="level3">
<h3 class="anchored" data-anchor-id="section-2">2021</h3>
<ul>
<li><p><font size="+1"> <strong>Kim, H.J., Wang, K.Y., Chen, C., Lin, Y.</strong>, Tam, P., Lin, D.M., <strong>Yang, J.Y.H. and Yang, P.</strong> (2021) Uncovering cell identity through differential stability with Cepo. <em>Nature Computational Science</em>. [<a href="https://PYangLab.github.io/Cepo">package</a>][<a href="https://doi.org/10.1038/s43588-021-00172-2">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y., Yang, P., and Yang, J.Y.H.</strong> (2021) A benchmark study of simulation methods for single-cell RNA sequencing data. <em>Nature Communications</em>. [<a href="https://github.com/SydneyBioX/SimBench">package</a>][<a href="https://doi.org/10.1038/s41467-021-27130-w">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Chan, A.</strong>, Jiang W., Blyth E., <strong>Yang, J.Y.H., and Patrick, E.</strong> (2021) treekoR: identifying cellular-to-phenotype associations by elucidating hierarchical relationships in high-dimensional cytometry data. <em>Genome Biology</em>. [<a href="http://bioconductor.org/packages/release/bioc/html/treekoR.html">package</a>][<a href="https://doi.org/10.1186/s13059-021-02526-5">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yang, P.</strong>, Huang, H., and Liu, C. (2021) Feature selection revisited in the single-cell era. <em>Genome Biology</em>. [<a href="https://doi.org/10.1186/s13059-021-02544-3">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Baharlou, H., Canete, N. P.</strong>, Bertram, K. M., Sandgren, K. J., Cunningham, A. L., Harman, A. N., and <strong>Patrick, E.</strong> (2021). AFid: a tool for automated identification and exclusion of autofluorescent objects from microscopy images. <em>Bioinformatics</em>, 37(4):559–567. [<a href="https://doi.org/10.1093/bioinformatics/btaa780">paper</a>]</font></p></li>
</ul>
</section>
<section id="section-3" class="level3">
<h3 class="anchored" data-anchor-id="section-3">2020</h3>
<ul>
<li><p><font size="+1"> <strong>Ghazanfar, S., Lin, Y.</strong>, Su, X., Lin, D.M., <strong>Patrick, E.,</strong> Han, Z.G., Marioni, J.C., and <strong>Yang, J.Y.H.</strong> (2020) Investigating higher-order interactions in single-cell data with scHOT. <em>Nature Methods</em>. [<a href="https://shazanfar.github.io/scHOT">package</a>][<a href="https://doi.org/10.1038/s41592-020-0885-x">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y., Cao, Y., Kim, H.J.,</strong> Salim, A., Speed, T., Lin, D.M., <strong>Yang, P. and Yang, J.Y.H.</strong> (2020) scClassify: sample size estimation and multiscale classification of cells using single and multiple reference. <em>Molecular Systems Biology</em>. [<a href="https://sydneybiox.github.io/scClassify">package</a>][<a href="http://shiny.maths.usyd.edu.au/scClassify">shiny</a>][<a href="https://doi.org/10.15252/msb.20199389">paper</a>] </font></p></li>
<li><p><font size="+1"> <strong>Kim, H.J., Lin, Y., Geddes, T.A., Yang, J.Y.H. and Yang, P.</strong> (2020) CiteFuse enables multi-modal analysis of CITE-seq data. <em>Bioinformatics</em>. [<a href="https://sydneybiox.github.io/CiteFuse">package</a>][<a href="http://shiny.maths.usyd.edu.au/CiteFuse">shiny</a>][<a href="https://doi.org/10.1093/bioinformatics/btaa282">paper</a>]</font></p></li>
<li><p><font size="+1"> Patrick, R., Humphreys, D.T., Janbandhu, V., Oshlack, A., Ho, J.W.K., Harvey, R.P., <strong>Lo, K.K.</strong> (2020) Sierra: discovery of differential transcript usage from polyA-captured single-cell RNA-seq data. <em>Genome Biology</em>. [<a href="https://github.com/VCCRI/Sierra">package</a>][<a href="https://doi.org/10.1186/s13059-020-02071-7">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Patrick, E.</strong>, Taga, M., Ergun, A., Ng, B., Casazza, W., Cimpean, M., Yung, C., Schneider, J. A., Bennett, D. A., Gaiteri, C., Jager, P. L. D., Bradshaw, E. M., and Mostafavi, S. (2020). Deconvolving the contributions of cell-type heterogeneity on cortical gene expression. <em>PLOS Computational Biology</em>, 16(8):e1008120.[<a href="https://doi.org/10.1371/journal.pcbi.1008120">paper</a>]</font></p></li>
</ul>
</section>
<section id="section-4" class="level3">
<h3 class="anchored" data-anchor-id="section-4">2019</h3>
<ul>
<li><p><font size="+1"> <strong>Lin, Y., Ghazanfar, S., Wang, K.Y.,</strong> Gagnon-Bartsch, J.A., <strong>Lo, K.K.,</strong> Su, X., Han, Z.G., <strong>Ormerod, J.T., Speed, T.P., Yang, P. and Yang, J.Y.H.</strong> (2019) scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets. <em>Proceedings of the National Academy of Sciences</em>. [<a href="https://sydneybiox.github.io/scMerge">package</a>] [<a href="https://doi.org/10.1073/pnas.1820006116">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Geddes, T., Kim, T.,</strong> Nan, L., Burchfield, J., <strong>Yang, J.Y.H.,</strong> Tao, D. and <strong>Yang, P.</strong> (2019) Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis. <em>BMC Bioinformatics</em>. [<a href="https://github.com/gedcom/scCCESS">package</a>] [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y., Lin, Y., Ormerod, J., Yang, P., Yang, J.Y.H. &amp; Lo, K.</strong> (2019) scDC: single cell differential composition analysis. <em>BMC Bioinformatics</em>. [<a href="https://sydneybiox.github.io/scDC">package</a>] [<a href="https://doi.org/10.1186/s12859-019-3211-9">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Kim, T., Lo, K., Geddes, T., Kim, H.J., Yang, J.Y.H. &amp; Yang, P.</strong> (2019) scReClassify: post hoc cell type classification of single-cell RNA-seq data. <em>BMC Genomics</em>. [<a href="https://github.com/SydneyBioX/scReClassify">package</a>] [<a href="https://doi.org/10.1186/s12864-019-6305-x">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y., Ghazanfar, S., Strbenac, D., Wang, A., Patrick, E.,</strong> Lin, D., Speed, T., <strong>Yang, J.Y.H. &amp; Yang, P.</strong> (2019) Evaluating stably expressed genes in single cells. <em>GigaScience</em>. [<a href="https://github.com/SydneyBioX/scMerge">package</a>] [<a href="https://doi.org/10.1093/gigascience/giz106">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Baharlou, H., Canete, N. P.,</strong> Cunningham, A. L., Harman, A. N., and <strong>Patrick, E.</strong> (2019). Mass Cytometry Imaging for the Study of Human Diseases—Applications and Data Analysis Strategies. <em>Frontiers in Immunology</em>, 10.[<a href="https://doi.org/10.3389/fimmu.2019.02657">paper</a>]</font></p></li>
</ul>
</section>
<section id="section-5" class="level3">
<h3 class="anchored" data-anchor-id="section-5">2018</h3>
<ul>
<li><font size="+1"> <strong>Kim, T., Chen, I., Lin, Y., Wang, A., Yang, J.Y.H. and Yang, P.</strong> (2018) Impact of similarity metrics on single-cell RNA-seq data clustering. <em>Briefings in Bioinformatics</em>. [<a href="https://taiyunkim.github.io/scClustBench">package</a>][<a href="https://doi.org/10.1093/bib/bby076">paper</a>]</font></li>
</ul>
</section>
<section id="section-6" class="level3">
<h3 class="anchored" data-anchor-id="section-6">2016</h3>
<ul>
<li><font size="+1"> <strong>Ghazanfar, S.,</strong> Bisogni, A., <strong>Ormerod, J.T.,</strong>, Lin, D.M., and <strong>Yang, J.Y.H.</strong> (2016) Integrated single cell data analysis reveals cell specific networks and novel coactivation markers. <em>BMC Systems Biology</em>. [<a href="https://doi.org/10.1186/s12918-016-0370-4">paper</a>]</font></li>
</ul>
</section>
</section>
<section id="applications" class="level2">
<h2 class="anchored" data-anchor-id="applications"><strong>Applications</strong></h2>
<ul>
<li><p><font size="+1"> <strong>Kim, H.J.,</strong> O’Hara-Wright, M., <strong>Kim, D.,</strong> Loi, T. H., Lim, B. Y., Jamieson, R. V., … &amp; <strong>Yang, P.</strong> (2023). Comprehensive characterization of fetal and mature retinal cell identity to assess the fidelity of retinal organoids. Stem Cell Reports, 18(1), 175-189. [<a href="https://www.sciencedirect.com/science/article/pii/S2213671122005914">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y.,</strong> Loo, L., <strong>Tran, A.,</strong> Lin, D. M., Moreno, C., Hesselson, D., Neely G.G., &amp; <strong>Yang, J.Y.</strong> (2022). Scalable workflow for characterization of cell-cell communication in COVID-19 patients. <em>PLOS Computational Biology</em>, 18(10), e1010495. [<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010495">paper</a>]</font></p></li>
<li><p><font size="+1"> Su, X., Zhao, L.N., Shi, Y., Zhang, R., Long, Q., Bai, S., Luo, Q., <strong>Lin Y.,</strong>, Zou, X., <strong>Ghazanfar, S.,</strong>, Tao, Kun., Yang, G., Wang, L., He, K.Y., Cui, X., He, J., Wu, J.X., Han, B., Wang, N., Li, X., <strong>Yang, P.</strong>, Hou, S., Sun, J., <strong>Yang, J.Y.H.,</strong> Chen, J. and Han, Z.G. (2021) Single-cell and single-variant resolution analysis of clonal evolution in human liver cancer. <em>Journal of Hematology &amp; Oncology</em>. [<a href="https://doi.org/10.1101/2020.12.30.424907">paper</a>]</font></p></li>
<li><p><font size="+1"> Singh, R., Cottle, L., Loudovaris, T., <strong>Xiao, D., Yang. P.,</strong> Thomas, H., Kebede, A. M., and Thorn, P. (2020) Enhanced structure and function of human pluripotent stem cell-derived beta-cells cultured on extracellular matrix. <em>Stem cells translational medicine</em>. [<a href="https://doi.org/10.1002/sctm.20-0224">paper</a>]</font></p></li>
<li><p><font size="+1"> Su, X., Long, Q., Bo, J., Shi, Y., Zhao, L.N., <strong>Lin Y.,</strong> Luo, Q., <strong>Ghazanfar, S.</strong>, Zhang, C., Liu, Q., Wang, L., He, K., He, J., Cui, X., <strong>Yang, J.Y.H.</strong>, Han, Z.G., Yang, G., and Sha, J.J. (2020) Mutational and transcriptomic landscapes of a rare human prostate basal cell carcinoma. <em>The Prostate</em>. [<a href="https://doi.org/10.1002/pros.23965">paper</a>]</font></p></li>
<li><p><font size="+1"> McGuire, H. M., Rizzetto, S., Withers, B. P., Clancy, L. E., Avdic, S., Stern, L., <strong>Patrick, E.</strong>, Groth, B. F. d.&nbsp;S., Slobedman, B., Gottlieb, D. J., Luciani, F., and Blyth, E. (2020). Mass cytometry reveals immune signatures associated with cytomegalovirus (CMV) control in recipients of allogeneic haemopoietic stem cell transplant and CMV-specific T cells. <em>Clinical &amp; Translational Immunology</em>, 9(7):e1149.</font></p></li>
<li><p><font size="+1"> Jimenez Vera, E., Chew, Y. V., Nicholson, L., Burns, H., Anderson, P., Chen, H.-T., Williams, L., Keung, K., Zanjani, N. T., Dervish, S., <strong>Patrick, E.</strong>, Wang, X. M., Yi, S., Hawthorne, W., Alexander, S., O’Connell, P. J., and Hu, M. (2019). Standardisation of flow cytometry for whole blood immunopheno-typing of islet transplant and transplant clinical trial recipients. <em>PLoS ONE</em>, 14(5).</font></p></li>
<li><p><font size="+1"> He J., <strong>Lin Y.,</strong>, Su, X., Luo, Q., <strong>Ghazanfar, S., Yang, J.Y.H.</strong> and Han,Z.G. (2019) Single-Cell RNA-Seq Reveals Naïve B cells Associated with Better Prognosis of HCC. <em>Biorxiv</em>. [<a href="https://doi.org/10.1101/731935">paper</a>]</font></p></li>
<li><p><font size="+1"> Su, X., Shi, Y., Zou, X., Lu, Z.N., Xie, G., <strong>Yang, J.Y.H.,</strong> Wu, C.C., Cui., X.F., He, K.Y., Luo, Q., Qu, Y.L., Wang, N., Wang, L., Han, Z.G. (2017) Single-cell RNA-Seq analysis reveals dynamic trajectories during mouse liver development <em>BMC Genomics</em>. [<a href="https://doi.org/10.1186/s12864-017-4342-x">paper</a>]</font></p></li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>