<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>SCDNEY - Software</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./images/USydLogo.svg" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">SCDNEY</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html" rel="" target="">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html" rel="" target="">
 <span class="menu-text">About us</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./software.html" rel="" target="" aria-current="page">
 <span class="menu-text">Software</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Publications.html" rel="" target="">
 <span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./workshops.html" rel="" target="">
 <span class="menu-text">Problem-focused Workshops</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#moleculeexperiment" id="toc-moleculeexperiment" class="nav-link active" data-scroll-target="#moleculeexperiment">MoleculeExperiment</a></li>
  <li><a href="#simpleseg" id="toc-simpleseg" class="nav-link" data-scroll-target="#simpleseg">simpleSeg</a></li>
  <li><a href="#scmerge" id="toc-scmerge" class="nav-link" data-scroll-target="#scmerge">scMerge</a></li>
  <li><a href="#fusesom" id="toc-fusesom" class="nav-link" data-scroll-target="#fusesom">FuseSOM</a></li>
  <li><a href="#treekor" id="toc-treekor" class="nav-link" data-scroll-target="#treekor">treekoR</a></li>
  <li><a href="#scfeatures" id="toc-scfeatures" class="nav-link" data-scroll-target="#scfeatures">scFeatures</a></li>
  <li><a href="#schot" id="toc-schot" class="nav-link" data-scroll-target="#schot">scHOT</a></li>
  <li><a href="#spicyr" id="toc-spicyr" class="nav-link" data-scroll-target="#spicyr">spicyR</a></li>
  <li><a href="#statial" id="toc-statial" class="nav-link" data-scroll-target="#statial">Statial</a></li>
  <li><a href="#lisaclust" id="toc-lisaclust" class="nav-link" data-scroll-target="#lisaclust">lisaClust</a></li>
  <li><a href="#classifyr" id="toc-classifyr" class="nav-link" data-scroll-target="#classifyr">ClassifyR</a></li>
  <li><a href="#citefuse" id="toc-citefuse" class="nav-link" data-scroll-target="#citefuse">CiteFuse</a></li>
  <li><a href="#scclassify" id="toc-scclassify" class="nav-link" data-scroll-target="#scclassify">scClassify</a></li>
  <li><a href="#screclassify" id="toc-screclassify" class="nav-link" data-scroll-target="#screclassify">scReClassify</a></li>
  <li><a href="#stabmap" id="toc-stabmap" class="nav-link" data-scroll-target="#stabmap">StabMap</a></li>
  <li><a href="#cepo" id="toc-cepo" class="nav-link" data-scroll-target="#cepo">Cepo</a></li>
  <li><a href="#simbench" id="toc-simbench" class="nav-link" data-scroll-target="#simbench">SimBench</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Software</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>


<link rel="stylesheet" href="css/hexagons.css">

<div id="importedContent"></div>
    <script>
        fetch('https://raw.githubusercontent.com/SydneyBioX/packageHeader/main/scdneyHeader.html')
            .then(response => response.text())
            .then(htmlContent => {
                const importedContentDiv = document.getElementById('importedContent');
                importedContentDiv.innerHTML = htmlContent;
            })
            .catch(error => {
                console.error('Error fetching content:', error);
            });
    </script>
<style>
citation {
  font-size: 9pt;
}
</style>
<section id="moleculeexperiment" class="level2">
<h2 class="anchored" data-anchor-id="moleculeexperiment">MoleculeExperiment</h2>
<p><a href="https://sydneybiox.github.io/MoleculeExperiment/"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/MoleculeExperiment/MoleculeExperiment.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>MoleculeExperiment contains functions to create and work with objects from the new MoleculeExperiment class. We introduce this class for analysing molecule-based spatial transcriptomics data (e.g., Xenium by 10X, Cosmx SMI by Nanostring, and Merscope by Vizgen). This allows researchers to analyse spatial transcriptomics data at the molecule level, and to have standardised data formats accross vendors.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Peters Couto B, Robertson N, Patrick E, Ghazanfar S (2023). <em>MoleculeExperiment: Prioritising a molecule-level storage of Spatial Transcriptomics Data</em>. <a href="https://doi.org/10.18129/B9.bioc.MoleculeExperiment">doi:10.18129/B9.bioc.MoleculeExperiment</a>, R package version 1.0.2.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="simpleseg" class="level2">
<h2 class="anchored" data-anchor-id="simpleseg">simpleSeg</h2>
<p><a href="http://www.bioconductor.org/packages/release/bioc/html/simpleSeg.html"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/simpleSeg/SimpleSeg_Hex_standard.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>Image segmentation is the process of identifying the borders of individual objects (in this case cells) within an image. This allows for the features of cells such as marker expression and morphology to be extracted, stored and analysed. simpleSeg provides functionality for user friendly, watershed based segmentation on multiplexed cellular images in R based on the intensity of user specified protein marker channels. simpleSeg can also be used for the normalization of single cell data obtained from multiple images.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Canete N, Nicholls A, Patrick E (2023). <em>simpleSeg: A package to perform simple cell segmentation</em>. <a href="https://doi.org/10.18129/B9.bioc.simpleSeg">doi:10.18129/B9.bioc.simpleSeg</a>, R package version 1.2.5.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="scmerge" class="level2">
<h2 class="anchored" data-anchor-id="scmerge">scMerge</h2>
<p><a href="https://sydneybiox.github.io/scMerge/"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scMerge/scMerge.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>Like all gene expression data, single-cell data suffers from batch effects and other unwanted variations that makes accurate biological interpretations difficult. The scMerge method leverages factor analysis, stably expressed genes (SEGs) and (pseudo-) replicates to remove unwanted variations and merge multiple single-cell data. This package contains all the necessary functions in the scMerge pipeline, including the identification of SEGs, replication-identification methods, and merging of single-cell data.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Lin Y, Ghazanfar S, Wang K, Gagnon-Bartsch J, Lo K, Su X, Han Z, Ormerod J, Speed T, Yang P, Yang J (2019). “scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets.” <em>Proceedings of the National Academy of Sciences</em>. <a href="https://doi.org/10.1073/pnas.1820006116">doi:10.1073/pnas.1820006116</a>.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="fusesom" class="level2">
<h2 class="anchored" data-anchor-id="fusesom">FuseSOM</h2>
<p><a href="http://www.bioconductor.org/packages/release/bioc/html/FuseSOM.html"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/FuseSOM/fuseSOM_ai_standard.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>A correlation-based multiview self-organizing map for the characterization of cell types in highly multiplexed in situ imaging cytometry assays (<code>FuseSOM</code>) is a tool for unsupervised clustering. <code>FuseSOM</code> is robust and achieves high accuracy by combining a <code>Self Organizing Map</code> architecture and a <code>Multiview</code> integration of correlation based metrics. This allows FuseSOM to cluster highly multiplexed in situ imaging cytometry assays.</p>
<p><font size="-1"></font></p><font size="-1">
<p>&lt;0-length citation&gt;</p>
</font><p><font size="-1"></font></p>
</section>
<section id="treekor" class="level2">
<h2 class="anchored" data-anchor-id="treekor">treekoR</h2>
<p><a href="http://www.bioconductor.org/packages/release/bioc/html/treekoR.html"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/treekoR/treekoR.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>treekoR is a novel framework that aims to utilise the hierarchical nature of single cell cytometry data to find robust and interpretable associations between cell subsets and patient clinical end points. These associations are aimed to recapitulate the nested proportions prevalent in workflows inovlving manual gating, which are often overlooked in workflows using automatic clustering to identify cell populations. We developed treekoR to: Derive a hierarchical tree structure of cell clusters; quantify a cell types as a proportion relative to all cells in a sample (%total), and, as the proportion relative to a parent population (%parent); perform significance testing using the calculated proportions; and provide an interactive html visualisation to help highlight key results.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Chan A (2023). <em>treekoR: Cytometry Cluster Hierarchy and Cellular-to-phenotype Associations</em>. <a href="https://doi.org/10.18129/B9.bioc.treekoR">doi:10.18129/B9.bioc.treekoR</a>, R package version 1.8.0.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="scfeatures" class="level2">
<h2 class="anchored" data-anchor-id="scfeatures">scFeatures</h2>
<p><a href="http://www.bioconductor.org/packages/release/bioc/html/scFeatures.html"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scFeatures/sticker_version2.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>scFeatures constructs multi-view representations of single-cell and spatial data. scFeatures is a tool that generates multi-view representations of single-cell and spatial data through the construction of a total of 17 feature types. These features can then be used for a variety of analyses using other software in Biocondutor.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Cao,Y., Lin,Y., Patrick,E., Yang,P., Yang,J.Y.H. &amp; (2022). “scFeatures: multi-view representations of single-cell and spatial data for disease outcome prediction.” <em>Bioinformatics</em>, <b>38</b>(20), 4745-4753. ISSN 1367-4803, <a href="https://doi.org/10.1093/bioinformatics/btac590">doi:10.1093/bioinformatics/btac590</a>.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="schot" class="level2">
<h2 class="anchored" data-anchor-id="schot">scHOT</h2>
<p><a href="https://bioconductor.org/packages/release/bioc/html/scHOT.html"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scHOT/scHOT_hex.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>Single cell Higher Order Testing (scHOT) is an R package that facilitates testing changes in higher order structure of gene expression along either a developmental trajectory or across space. scHOT is general and modular in nature, can be run in multiple data contexts such as along a continuous trajectory, between discrete groups, and over spatial orientations; as well as accommodate any higher order measurement such as variability or correlation. scHOT meaningfully adds to first order effect testing, such as differential expression, and provides a framework for interrogating higher order interactions from single cell data.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Ghazanfar S, Lin Y (2023). <em>scHOT: single-cell higher order testing</em>. <a href="https://doi.org/10.18129/B9.bioc.scHOT">doi:10.18129/B9.bioc.scHOT</a>, R package version 1.12.0.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="spicyr" class="level2">
<h2 class="anchored" data-anchor-id="spicyr">spicyR</h2>
<p><a href="https://ellispatrick.github.io/spicyR/"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/spicyR/spicyR_standard.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>The spicyR package provides a framework for performing inference on changes in spatial relationships between pairs of cell types for cell-resolution spatial omics technologies. spicyR consists of three primary steps: (i) summarizing the degree of spatial localization between pairs of cell types for each image; (ii) modelling the variability in localization summary statistics as a function of cell counts and (iii) testing for changes in spatial localizations associated with a response variable.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Canete N, Iyengar S, Ormerod J, Baharlou H, Harman A, Patrick E (2022). “spicyR: spatial analysis of in situ cytometry data in R.” <em>Bioinformatics</em>, <b>38</b>(11), 3099–3105. <a href="https://doi.org/10.1093/bioinformatics/btac268">doi:10.1093/bioinformatics/btac268</a>.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="statial" class="level2">
<h2 class="anchored" data-anchor-id="statial">Statial</h2>
<p><a href="https://sydneybiox.github.io/Statial/"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/Statial/statial.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>Statial is a suite of functions for identifying changes in cell state. The functionality provided by Statial provides robust quantification of cell type localisation which are invariant to changes in tissue structure. In addition to this Statial uncovers changes in marker expression associated with varying levels of localisation. These features can be used to explore how the structure and function of different cell types may be altered by the agents they are surrounded with.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Ameen F, Iyengar S, Ghazanfar S, Patrick E (2023). <em>Statial: A package to identify changes in cell state relative to spatial associations</em>. <a href="https://doi.org/10.18129/B9.bioc.Statial">doi:10.18129/B9.bioc.Statial</a>, R package version 1.2.4.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="lisaclust" class="level2">
<h2 class="anchored" data-anchor-id="lisaclust">lisaClust</h2>
<p><a href="https://ellispatrick.github.io/lisaClust/"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/lisaClust/lisaClust_sticker_ai_standard.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>lisaClust provides a series of functions to identify and visualise regions of tissue where spatial associations between cell-types is similar. This package can be used to provide a high-level summary of cell-type colocalization in multiplexed imaging data that has been segmented at a single-cell resolution.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Patrick E, Canete N (2023). <em>lisaClust: lisaClust: Clustering of Local Indicators of Spatial Association</em>. <a href="https://doi.org/10.18129/B9.bioc.lisaClust">doi:10.18129/B9.bioc.lisaClust</a>, R package version 1.8.1.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="classifyr" class="level2">
<h2 class="anchored" data-anchor-id="classifyr">ClassifyR</h2>
<p><a href="https://sydneybiox.github.io/ClassifyR/"><img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/ClassifyR/classifyR.png?raw=true" align="right" style="height: 100px; border: 0px"></a></p>
<p>The software formalises a framework for classification and survival model evaluation in R. There are four stages; Data transformation, feature selection, model training, and prediction. The requirements of variable types and variable order are fixed, but specialised variables for functions can also be provided. The framework is wrapped in a driver loop that reproducibly carries out a number of cross-validation schemes. Functions for differential mean, differential variability, and differential distribution are included. Additional functions may be developed by the user, by creating an interface to the framework.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Strbenac D, Mann GJ, Ormerod JT, Yang JYH (2015). “ClassifyR: an R package for performance assessment of classification with applications to transcriptomics.” <em>Bioinformatics</em>, <b>31</b>(11), 1851-1853.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="citefuse" class="level2">
<h2 class="anchored" data-anchor-id="citefuse">CiteFuse</h2>
<p><a href="https://sydneybiox.github.io/CiteFuse/"><img src="https://raw.githubusercontent.com/SydneyBioX/SydneyBioXStickers/main/CiteFuse/CiteFuse.png" align="right" style="height: 100px; border: 0px"></a></p>
<p>CiteFuse pacakage implements a suite of methods and tools for CITE-seq data from pre-processing to integrative analytics, including doublet detection, network-based modality integration, cell type clustering, differential RNA and protein expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualisation of the analyses.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Kim HJ, Lin Y, Geddes TA, Yang P, Yang JYH (2020). “CiteFuse enables multi-modal analysis of CITE-seq data.” <em>Bioinformatics</em>, <b>36</b>(14), 4137–4143.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="scclassify" class="level2">
<h2 class="anchored" data-anchor-id="scclassify">scClassify</h2>
<p><a href="https://sydneybiox.github.io/scClassify/"><img src="https://raw.githubusercontent.com/SydneyBioX/SydneyBioXStickers/main/scClassify/scClassify.png" align="right" style="height: 100px; border: 0px"></a></p>
<p>scClassify is a multiscale classification framework for single-cell RNA-seq data based on ensemble learning and cell type hierarchies, enabling sample size estimation required for accurate cell type classification and joint classification of cells using multiple references.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Lin Y, Cao Y, Kim HJ, Salim A, Speed TP, Lin DM, Yang P, Yang JYH (2020). “scClassify: sample size estimation and multiscale classification of cells using single and multiple reference.” <em>Molecular systems biology</em>, <b>16</b>(6), e9389.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="screclassify" class="level2">
<h2 class="anchored" data-anchor-id="screclassify">scReClassify</h2>
<p><a href="https://github.com/SydneyBioX/scReClassify"><img src="https://raw.githubusercontent.com/SydneyBioX/SydneyBioXStickers/main/scReClassify/scReClassify.png" align="right" style="height: 100px; border: 0px"></a></p>
<p>A post hoc cell type classification tool to fine-tune cell type annotations generated by any cell type classification procedure with semi-supervised learning algorithm AdaSampling technique. The current version of scReClassify supports Support Vector Machine and Random Forest as a base classifier.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Kim T, Lo K, Geddes TA, Kim HJ, Yang JYH, Yang P (2019). “scReClassify: post hoc cell type classification of single-cell RNA-seq data.” <em>BMC Genomics</em>, <b>20</b>(913).
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="stabmap" class="level2">
<h2 class="anchored" data-anchor-id="stabmap">StabMap</h2>
<p><a href="https://marionilab.github.io/StabMap/"><img src="https://raw.githubusercontent.com/MarioniLab/StabMap/main/man/figures/hex.png" align="right" style="height: 100px; border: 0px"></a></p>
<p>StabMap performs single cell mosaic data integration by first building a mosaic data topology, and for each reference dataset, traverses the topology to project and predict data onto a common embedding. Mosaic data should be provided in a list format, with all relevant features included in the data matrices within each list object. The output of stabMap is a joint low-dimensional embedding taking into account all available relevant features. Expression imputation can also be performed using the StabMap embedding and any of the original data matrices for given reference and query cell lists.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Ghazanfar S (2023). <em>StabMap: Stabilised mosaic single cell data integration using unshared features</em>. R package version 0.1.8.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="cepo" class="level2">
<h2 class="anchored" data-anchor-id="cepo">Cepo</h2>
<p><a href="https://pyanglab.github.io/Cepo/"><img src="https://raw.githubusercontent.com/PYangLab/Cepo/main/man/figures/Cepo_logo.png" align="right" style="height: 100px; border: 0px"></a></p>
<p>Defining the identity of a cell is fundamental to understand the heterogeneity of cells to various environmental signals and perturbations. We present Cepo, a new method to explore cell identities from single-cell RNA-sequencing data using differential stability as a new metric to define cell identity genes. Cepo computes cell-type specific gene statistics pertaining to differential stable gene expression.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Kim H, Wang K (2023). <em>Cepo: Cepo for the identification of differentially stable genes</em>. <a href="https://doi.org/10.18129/B9.bioc.Cepo">doi:10.18129/B9.bioc.Cepo</a>, R package version 1.6.0.
</p>
</font><p><font size="-1"></font></p>
</section>
<section id="simbench" class="level2">
<h2 class="anchored" data-anchor-id="simbench">SimBench</h2>
<p><a href="https://github.com/SydneyBioX/SimBench"><img src="https://raw.githubusercontent.com/SydneyBioX/SydneyBioXStickers/main/SimBench/simbench_sticker.png" align="right" style="height: 100px; border: 0px"></a></p>
<p>The SimBench package is designed for benchmarking simulation methods based on two key aspects of accuracy of data properties estimation and ability to retain biological signals. It contains functions for comparing simulated data obtained from simulation methods and real data that was used as the reference input into the simulation methods on the two aspects.</p>
<p><font size="-1"></font></p><font size="-1">
<p>
Cao Y, Yang P, Yang J (2021). <em>SimBench: SimBench: benchmarking simulation methods</em>. R package version 0.99.1.
</p>
</font><p><font size="-1"></font></p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>