<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="scdney">
<title>scdney packages • scdney</title>
<script src="../../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="../../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../../pkgdown.js"></script><meta property="og:title" content="scdney packages">
<meta property="og:description" content="scdney">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-dark"><div class="container">
    
    <a class="navbar-brand me-2" href="../../index.html">scdney</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.2.4</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="active nav-item">
  <a class="nav-link" href="../../articles/website/package.html">
    <span class="fa fa-laptop-code fa-lg"></span>
     
    Packages
  </a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../../articles/website/publication.html">
    <span class="fa fa-book fa-lg"></span>
     
    Publications
  </a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/SydneyBioX/scdney">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>scdney packages</h1>
            
      
      
      <div class="d-none name"><code>package.Rmd</code></div>
    </div>

    
    
<p><font size="+1"> <code>scdney</code> include the packages for
single-cell data analysis for data integration (<a href="https://github.com/SydneyBioX/scMerge" class="external-link">scMerge</a>), cell type
identification (<a href="https://taiyunkim.github.io/scClustBench" class="external-link">scClust</a>, <a href="https://github.com/SydneyBioX/scClassify" class="external-link">scClassify</a>, <a href="https://github.com/SydneyBioX/scReClassify" class="external-link">scReClassify</a>),
multi-modal data analysis (<a href="https://github.com/SydneyBioX/scReClassify" class="external-link">CiteFuse</a>), higher
order testing (<a href="https://github.com/shazanfar/scHOT" class="external-link">scHOT</a>),
differential stable genes analysis (<a href="https://github.com/PYangLab/Cepo" class="external-link">Cepo</a>) and differential
composition analysis (<a href="https://github.com/SydneyBioX/scDC" class="external-link">scDC</a>). </font></p>
<div class="section level2">
<h2 id="data-integration">
<strong>Data integration</strong><a class="anchor" aria-label="anchor" href="#data-integration"></a>
</h2>
<div class="section level3">
<h3 id="scmerge">
<strong>scMerge</strong><a class="anchor" aria-label="anchor" href="#scmerge"></a>
</h3>
<div class="row">
<div class="col-md-7" markdown="1">
Single-cell RNA-sequencing (scRNA-seq) profiling has exploded in recent
years and enabled new biological knowledge to be discovered at the
single-cell level. Successful and flexible integration of scRNA-Seq
datasets from multiple sources promises to be an effective avenue to
obtain further biological insights. This study presents a comprehensive
approach to integration for scRNA-seq data analysis. It addresses the
challenges involved in successful integration of scRNA-seq datasets by
using the knowledge of genes that appear not to change across all
samples and a robust algorithm to infer pseudoreplicates between
datasets. This information is then consolidated into a single-factor
model that enables tailored incorporation of prior knowledge. The
effectiveness of scMerge is demonstrated by extensive comparison with
other approaches.
</div>
<div class="col-md-5" markdown="1">
<!-- ![Alt Text](../img/folder/blah.jpg) -->
<img src="../../img/scMerge_fig.jpg" align="right" width="500">
</div>
</div>
<ul>
<li>
<strong>Github</strong>: <a href="https://github.com/SydneyBioX/scMerge" class="external-link uri">https://github.com/SydneyBioX/scMerge</a>
</li>
<li>
<strong>Reference</strong>: Lin, Y., Ghazanfar, S., Wang, K.Y.,
Gagnon-Bartsch, J.A., Lo, K.K., Su, X., Han, Z.G., Ormerod, J.T., Speed,
T.P., Yang, P. and Yang, J.Y.H. (2019) scMerge leverages factor
analysis, stable expression, and pseudoreplication to merge multiple
single-cell RNA-seq datasets. <em>Proceedings of the National Academy of
Sciences</em>.</li>
</ul>
</div>
</div>
<div class="section level2">
<h2 id="cell-type-identification">
<strong>Cell type identification</strong><a class="anchor" aria-label="anchor" href="#cell-type-identification"></a>
</h2>
<div class="section level3">
<h3 id="scclassify">
<strong>scClassify</strong><a class="anchor" aria-label="anchor" href="#scclassify"></a>
</h3>
<p><!-- ![Alt Text](../img/folder/blah.jpg) -->
<img src="../../img/scClassify_fig.jpg" align="left" width="450" hspace="20" vspace="10">
Automated cell type identification is a key computational challenge in
single‐cell RNA‐sequencing (scRNA‐seq) data. To capitalise on the large
collection of well‐annotated scRNA‐seq datasets, we developed
scClassify, a multiscale classification framework based on ensemble
learning and cell type hierarchies constructed from single or multiple
annotated datasets as references. scClassify enables the estimation of
sample size required for accurate classification of cell types in a cell
type hierarchy and allows joint classification of cells when multiple
references are available. We show that scClassify consistently performs
better than other supervised cell type classification methods across 114
pairs of reference and testing data, representing a diverse combination
of sizes, technologies and levels of complexity, and further demonstrate
the unique components of scClassify through simulations and compendia of
experimental datasets. Finally, we demonstrate the scalability of
scClassify on large single‐cell atlases and highlight a novel
application of identifying subpopulations of cells from the Tabula Muris
data that were unidentified in the original publication. Together,
scClassify represents state‐of‐the‐art methodology in automated cell
type identification from scRNA‐seq data.</p>
<ul>
<li>
<strong>Github</strong>: <a href="https://github.com/SydneyBioX/scClassify" class="external-link uri">https://github.com/SydneyBioX/scClassify</a>
</li>
<li>
<strong>Reference</strong>: Lin, Y., Cao, Y., Kim, H.J., Salim, A.,
Speed, T., Lin, D.M., Yang, P. and Yang, J.Y.H. (2020) scClassify:
sample size estimation and multiscale classification of cells using
single and multiple reference. <em>Molecular Systems Biology</em>.</li>
</ul>
</div>
<div class="section level3">
<h3 id="screclassify">
<strong>scReClassify</strong><a class="anchor" aria-label="anchor" href="#screclassify"></a>
</h3>
<p><img src="../../img/scReClassify.jpg" align="right" width="450" hspace="20" vspace="10">
Single-cell RNA-sequencing (scRNA-seq) is a fast emerging technology
allowing global transcriptome profiling on the single cell level. Cell
type identification from scRNA-seq data is a critical task in a variety
of research such as developmental biology, cell reprogramming, and
cancers. Typically, cell type identification relies on human inspection
using a combination of prior biological knowledge (e.g. marker genes and
morphology) and computational techniques (e.g. PCA and clustering). Due
to the incompleteness of our current knowledge and the subjectivity
involved in this process, a small amount of cells may be subject to
mislabelling. Here, we propose a semi-supervised learning framework,
named scReClassify, for ‘post hoc’ cell type identification from
scRNA-seq datasets. Starting from an initial cell type annotation with
potentially mislabelled cells, scReClassify first performs dimension
reduction using PCA and next applies a semi-supervised learning method
to learn and subsequently reclassify cells that are likely mislabelled
initially to the most probable cell types. By using both simulated and
real-world experimental datasets that profiled various tissues and
biological systems, we demonstrate that scReClassify is able to
accurately identify and reclassify misclassified cells to their correct
cell types. scReClassify can be used for scRNA-seq data as a post hoc
cell type classification tool to fine-tune cell type annotations
generated by any cell type classification procedure.</p>
<ul>
<li>
<strong>Github</strong>: <a href="https://github.com/SydneyBioX/scReClassify" class="external-link uri">https://github.com/SydneyBioX/scReClassify</a>
</li>
<li>
<strong>Reference</strong>: Kim, T., Lo, K., Geddes, T., Kim, H.J.,
Yang, J.Y.H. &amp; Yang, P. (2019) scReClassify: post hoc cell type
classification of single-cell RNA-seq data. <em>BMC Genomics</em>.</li>
</ul>
</div>
<div class="section level3">
<h3 id="scclust">
<strong>scClust</strong><a class="anchor" aria-label="anchor" href="#scclust"></a>
</h3>
<p><img src="../../img/scClust.jpg" align="left" width="450" hspace="20" vspace="10">
Advances in high-throughput sequencing on single-cell gene expressions
have enabled transcriptome profiling on individual cells from complex
samples. A common goal in scRNA-seq data analysis is to discover and
characterise cell types, typically through clustering methods. The
quality of the clustering therefore plays a critical role in biological
discovery. While numerous clustering algorithms have been proposed for
scRNA-seq data, fundamentally they all rely on a similarity metric for
categorising individual cells. Although several studies have compared
the performance of various clustering algorithms for scRNA-seq data,
currently there is no benchmark of different similarity metrics and
their influence on scRNA-seq data clustering. Here, we compared a panel
of similarity metrics on clustering a collection of annotated scRNA-seq
datasets. Within each dataset, a stratified subsampling procedure was
applied and an array of evaluation measures was employed to assess the
similarity metrics. This produced a highly reliable and reproducible
consensus on their performance assessment. Overall, we found that
correlation-based metrics (e.g. Pearson’s correlation) outperformed
distance-based metrics (e.g. Euclidean distance). To test if the use of
correlation-based metrics can benefit the recently published clustering
techniques for scRNA-seq data, we modified a state-of-the-art
kernel-based clustering algorithm (SIMLR) using Pearson’s correlation as
a similarity measure and found significant performance improvement over
Euclidean distance on scRNA-seq data clustering. These findings
demonstrate the importance of similarity metrics in clustering scRNA-seq
data and highlight Pearson’s correlation as a favourable choice. Further
comparison on different scRNA-seq library preparation protocols suggests
that they may also affect clustering performance.</p>
<ul>
<li>
<strong>Github</strong>: <a href="https://taiyunkim.github.io/scClustBench" class="external-link uri">https://taiyunkim.github.io/scClustBench</a>
</li>
<li>
<strong>Reference</strong>: Kim, T., Chen, I., Lin, Y., Wang, A.,
Yang, J.Y.H. and Yang, P. (2018) Impact of similarity metrics on
single-cell RNA-seq data clustering. <em>Briefings in
Bioinformatics</em>.</li>
</ul>
</div>
</div>
<div class="section level2">
<h2 id="multi-modal-data-analysis">
<strong>Multi-modal data analysis</strong><a class="anchor" aria-label="anchor" href="#multi-modal-data-analysis"></a>
</h2>
<div class="section level3">
<h3 id="citefuse">
<strong>CiteFuse</strong><a class="anchor" aria-label="anchor" href="#citefuse"></a>
</h3>
<p><!-- ![Alt Text](../img/folder/blah.jpg) -->
<img src="../../img/CiteFuse_fig.png" align="left" width="450" hspace="20" vspace="10">
Multi-modal profiling of single cells represents one of the latest
technological advancements in molecular biology. Among various
single-cell multi-modal strategies, cellular indexing of transcriptomes
and epitopes by sequencing (CITE-seq) allows simultaneous quantification
of two distinct species: RNA and cell-surface proteins. Here, we
introduce CiteFuse, a streamlined package consisting of a suite of tools
for doublet detection, modality integration, clustering, differential
RNA and protein expression analysis, antibody-derived tag evaluation,
ligand–receptor interaction analysis and interactive web-based
visualization of CITE-seq data. We demonstrate the capacity of CiteFuse
to integrate the two data modalities and its relative advantage against
data generated from single-modality profiling using both simulations and
real-world CITE-seq data. Furthermore, we illustrate a novel doublet
detection method based on a combined index of cell hashing and
transcriptome data. Finally, we demonstrate CiteFuse for predicting
ligand–receptor interactions by using multi-modal CITE-seq data.
Collectively, we demonstrate the utility and effectiveness of CiteFuse
for the integrative analysis of transcriptome and epitope profiles from
CITE-seq data.</p>
<ul>
<li>
<strong>Github</strong>: <a href="https://github.com/SydneyBioX/CiteFuse" class="external-link uri">https://github.com/SydneyBioX/CiteFuse</a>
</li>
<li>
<strong>Reference</strong>: Kim, H.J., Lin, Y., Geddes, T.A., Yang,
J.Y.H. and Yang, P. (2020) CiteFuse enables multi-modal analysis of
CITE-seq data. <em>Bioinformatics</em>.</li>
</ul>
</div>
</div>
<div class="section level2">
<h2 id="cell-marker-detection">
<strong>Cell marker detection</strong><a class="anchor" aria-label="anchor" href="#cell-marker-detection"></a>
</h2>
<div class="section level3">
<h3 id="cepo">
<strong>Cepo</strong><a class="anchor" aria-label="anchor" href="#cepo"></a>
</h3>
<p><!-- ![Alt Text](../img/folder/blah.jpg) -->
<img src="../../img/Cepo_logo_final_transparent.png" align="left" width="450" hspace="20" vspace="10">
Defining the identity of a cell is fundamental to understand the
heterogeneity of cells to various environmental signals and
perturbations. We present Cepo, a new method to explore cell identities
from single-cell RNA-sequencing data using differential stability as a
new metric to define cell identity genes. Cepo computes cell-type
specific gene statistics pertaining to differential stable gene
expression.</p>
<ul>
<li>
<strong>Github</strong>: <a href="https://github.com/PYangLab/Cepo" class="external-link uri">https://github.com/PYangLab/Cepo</a>
</li>
<li>
<strong>Reference</strong>: Kim, H.J., Wang, K., Chen, C., Lin, Y.,
Tam, P., Lin, D., Yang, J.Y.H. and Yang, P. (2020) Cepo uncovers cell
identity through differential stability. <em>bioRxiv</em>.</li>
</ul>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Sydney Precision Bioinformatics Alliance.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
