<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="scdney">
<title>Publications • scdney</title>
<script src="../../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="../../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../../pkgdown.js"></script><meta property="og:title" content="Publications">
<meta property="og:description" content="scdney">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-dark"><div class="container">
    
    <a class="navbar-brand me-2" href="../../index.html">scdney</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.2.4</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../../articles/website/package.html">
    <span class="fa fa-laptop-code fa-lg"></span>
     
    Packages
  </a>
</li>
<li class="active nav-item">
  <a class="nav-link" href="../../articles/website/publication.html">
    <span class="fa fa-book fa-lg"></span>
     
    Publications
  </a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/SydneyBioX/scdney">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Publications</h1>
            
      
      
      <div class="d-none name"><code>publication.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="methodology-and-tools">
<strong>Methodology and tools</strong><a class="anchor" aria-label="anchor" href="#methodology-and-tools"></a>
</h2>
<div class="section level3">
<h3 id="preprint">Preprint<a class="anchor" aria-label="anchor" href="#preprint"></a>
</h3>
<ul>
<li><p><font size="+1"> <strong>Lin, Y.</strong>, Loo, L., <strong>Tran,
A.</strong>, Moreno, C., Hesselson D., Neely, G., and <strong>Yang,
J.Y.H.</strong> (2020) Characterization of cell-cell communication in
COVID-19 patients. <em>Biorxiv</em>. [<a href="https://doi.org/10.1101/2020.12.30.424641" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y.</strong>, Cao, Y., Willie, E.,
Patrick, E., &amp; Yang, J. Y. H. (2022). Atlas-scale single-cell
multi-sample multi-condition data integration using scMerge2.
<em>BioRxiv</em>. [<a href="https://doi.org/10.1101/2022.12.08.519588" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yin, D.</strong>, <strong>Cao,
Y.</strong>, <strong>Chen, J.</strong>, Mak, C. L., Yu, K. H., Lin, Y.,
<strong>Ho, J.W.K.</strong> &amp; <strong>Yang, J.Y.H.</strong> (2022).
Covidscope: An atlas-scale COVID-19 resource for single-cell meta
analysis at sample and cell levels. <em>BioRxiv</em>. [<a href="https://doi.org/10.1101/2022.12.03.518997" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
<div class="section level3">
<h3 id="section">2023<a class="anchor" aria-label="anchor" href="#section"></a>
</h3>
<ul>
<li><p><font size="+1"> <strong>Patrick, E., Canete, N. P., Baharlou,
H., Iyengar, S. S.,</strong> Harman, A. N., Sutherland, G. T., and
<strong>Yang P.</strong> (2021) Spatial analysis for highly multiplexed
imaging data to identify tissue microenvironments. <em>Cytometry A</em>.
[<a href="http://bioconductor.org/packages/release/bioc/html/lisaClust.html" class="external-link">package</a>][<a href="http://shiny.maths.usyd.edu.au/lisaClust" class="external-link">shiny</a>][<a href="https://doi.org/10.1101/2021.08.16.456469" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y.</strong>, Ghazanfar, S., Yang,
P., &amp; <strong>Yang, J.Y.H.</strong> (2023). Benchmarking of
analytical combinations for COVID-19 outcome prediction using
single-cell RNA sequencing data. <em>Briefings in Bioinformatics</em>.
[<a href="https://doi.org/10.1093/bib/bbad159" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y.</strong>, <strong>Tran,
A.</strong>, <strong>Kim, H.</strong>, Robertson, N., Lin, Y., Torkel,
M., Yang, P., Patrick, E., <strong>Ghazanfar, S.</strong>, &amp;
<strong>Yang, J.Y.H.</strong> (2023). Thinking process templates for
constructing data stories with SCDNEY. <em>F1000 Research</em>. [<a href="https://doi.org/10.12688/f1000research.130623.1" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yu, L.</strong>, Cao, Y., Yang, J.Y.H.,
&amp; <strong>Yang, P</strong>. (2022). Benchmarking clustering
algorithms on estimating the number of cell types from single-cell
RNA-sequencing data. <em>Genome Biology</em>. [<a href="https://doi.org/10.1186/s13059-022-02622-0" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
<div class="section level3">
<h3 id="section-1">2022<a class="anchor" aria-label="anchor" href="#section-1"></a>
</h3>
<ul>
<li><p><font size="+1"> <strong>Canete, N. P., Iyengar, S. S.</strong>,
Wilmott. J. S., <strong>Ormerod, J. T.</strong>, Harman, A. N., and
<strong>Patrick, E.</strong>(2021) spicyR: Spatial analysis of in situ
cytometry data in R. <em>Bioinformatics</em>. [<a href="http://bioconductor.org/packages/release/bioc/html/spicyR.html" class="external-link">package</a>][<a href="http://shiny.maths.usyd.edu.au/spicyR/" class="external-link">shiny</a>][<a href="https://doi.org/10.1093/bioinformatics/btac268" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y., Lin, Y., Patrick, E., Yang, P.,
and Yang, J.Y.H.</strong> (2022) scFeatures: Multi-view representations
of single-cell and spatial data for disease outcome prediction.
<em>Bioinformatics</em>. [<a href="https://doi.org/10.1093/bioinformatics/btac590" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y.</strong>, Wu, T.Y., Wan, S.,
<strong>Yang, J.Y.H.,</strong> Wong, W.H., and <strong>Wang,
Y.X.R.</strong> (2022) scJoint: transfer learning for data integration
of single-cell RNA-seq and ATAC-seq. <em>Nature Biotechnology</em>. [<a href="https://github.com/SydneyBioX/scJoint" class="external-link">package</a>][<a href="https://doi.org/10.1038/s41587-021-01161-6" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Tran, A., Yang P., Yang, J.Y.H., and
Ormerod, J. T.</strong> (2022) scREMOTE: Using multimodal single cell
data to predict regulatory gene relationships and to build a
computational cell reprogramming model. <em>NAR Genomics and
Bioinformaitics</em>. [<a href="https://github.com/SydneyBioX/scREMOTE" class="external-link">package</a>][<a href="https://doi.org/10.1093/nargab/lqac023" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
<div class="section level3">
<h3 id="section-2">2021<a class="anchor" aria-label="anchor" href="#section-2"></a>
</h3>
<ul>
<li><p><font size="+1"> <strong>Kim, H.J., Wang, K.Y., Chen, C., Lin,
Y.</strong>, Tam, P., Lin, D.M., <strong>Yang, J.Y.H. and Yang,
P.</strong> (2021) Uncovering cell identity through differential
stability with Cepo. <em>Nature Computational Science</em>. [<a href="https://PYangLab.github.io/Cepo" class="external-link">package</a>][<a href="https://doi.org/10.1038/s43588-021-00172-2" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y., Yang, P., and Yang,
J.Y.H.</strong> (2021) A benchmark study of simulation methods for
single-cell RNA sequencing data. <em>Nature Communications</em>. [<a href="https://github.com/SydneyBioX/SimBench" class="external-link">package</a>][<a href="https://doi.org/10.1038/s41467-021-27130-w" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Chan, A.</strong>, Jiang W., Blyth E.,
<strong>Yang, J.Y.H., and Patrick, E.</strong> (2021) treekoR:
identifying cellular-to-phenotype associations by elucidating
hierarchical relationships in high-dimensional cytometry data.
<em>Genome Biology</em>. [<a href="http://bioconductor.org/packages/release/bioc/html/treekoR.html" class="external-link">package</a>][<a href="https://doi.org/10.1186/s13059-021-02526-5" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Yang, P.</strong>, Huang, H., and Liu,
C. (2021) Feature selection revisited in the single-cell era. <em>Genome
Biology</em>. [<a href="https://doi.org/10.1186/s13059-021-02544-3" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Baharlou, H., Canete, N. P.</strong>,
Bertram, K. M., Sandgren, K. J., Cunningham, A. L., Harman, A. N., and
<strong>Patrick, E.</strong> (2021). AFid: a tool for automated
identification and exclusion of autofluorescent objects from microscopy
images. <em>Bioinformatics</em>, 37(4):559–567. [<a href="https://doi.org/10.1093/bioinformatics/btaa780" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
<div class="section level3">
<h3 id="section-3">2020<a class="anchor" aria-label="anchor" href="#section-3"></a>
</h3>
<ul>
<li><p><font size="+1"> <strong>Ghazanfar, S., Lin, Y.</strong>, Su, X.,
Lin, D.M., <strong>Patrick, E.,</strong> Han, Z.G., Marioni, J.C., and
<strong>Yang, J.Y.H.</strong> (2020) Investigating higher-order
interactions in single-cell data with scHOT. <em>Nature Methods</em>.
[<a href="https://shazanfar.github.io/scHOT" class="external-link">package</a>][<a href="https://doi.org/10.1038/s41592-020-0885-x" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y., Cao, Y., Kim, H.J.,</strong>
Salim, A., Speed, T., Lin, D.M., <strong>Yang, P. and Yang,
J.Y.H.</strong> (2020) scClassify: sample size estimation and multiscale
classification of cells using single and multiple reference.
<em>Molecular Systems Biology</em>. [<a href="https://sydneybiox.github.io/scClassify" class="external-link">package</a>][<a href="http://shiny.maths.usyd.edu.au/scClassify" class="external-link">shiny</a>][<a href="https://doi.org/10.15252/msb.20199389" class="external-link">paper</a>] </font></p></li>
<li><p><font size="+1"> <strong>Kim, H.J., Lin, Y., Geddes, T.A., Yang,
J.Y.H. and Yang, P.</strong> (2020) CiteFuse enables multi-modal
analysis of CITE-seq data. <em>Bioinformatics</em>. [<a href="https://sydneybiox.github.io/CiteFuse" class="external-link">package</a>][<a href="http://shiny.maths.usyd.edu.au/CiteFuse" class="external-link">shiny</a>][<a href="https://doi.org/10.1093/bioinformatics/btaa282" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> Patrick, R., Humphreys, D.T., Janbandhu, V.,
Oshlack, A., Ho, J.W.K., Harvey, R.P., <strong>Lo, K.K.</strong> (2020)
Sierra: discovery of differential transcript usage from polyA-captured
single-cell RNA-seq data. <em>Genome Biology</em>. [<a href="https://github.com/VCCRI/Sierra" class="external-link">package</a>][<a href="https://doi.org/10.1186/s13059-020-02071-7" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Patrick, E.</strong>, Taga, M., Ergun,
A., Ng, B., Casazza, W., Cimpean, M., Yung, C., Schneider, J. A.,
Bennett, D. A., Gaiteri, C., Jager, P. L. D., Bradshaw, E. M., and
Mostafavi, S. (2020). Deconvolving the contributions of cell-type
heterogeneity on cortical gene expression. <em>PLOS Computational
Biology</em>, 16(8):e1008120.[<a href="https://doi.org/10.1371/journal.pcbi.1008120" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
<div class="section level3">
<h3 id="section-4">2019<a class="anchor" aria-label="anchor" href="#section-4"></a>
</h3>
<ul>
<li><p><font size="+1"> <strong>Lin, Y., Ghazanfar, S., Wang,
K.Y.,</strong> Gagnon-Bartsch, J.A., <strong>Lo, K.K.,</strong> Su, X.,
Han, Z.G., <strong>Ormerod, J.T., Speed, T.P., Yang, P. and Yang,
J.Y.H.</strong> (2019) scMerge leverages factor analysis, stable
expression, and pseudoreplication to merge multiple single-cell RNA-seq
datasets. <em>Proceedings of the National Academy of Sciences</em>. [<a href="https://sydneybiox.github.io/scMerge" class="external-link">package</a>] [<a href="https://doi.org/10.1073/pnas.1820006116" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Geddes, T., Kim, T.,</strong> Nan, L.,
Burchfield, J., <strong>Yang, J.Y.H.,</strong> Tao, D. and <strong>Yang,
P.</strong> (2019) Autoencoder-based cluster ensembles for single-cell
RNA-seq data analysis. <em>BMC Bioinformatics</em>. [<a href="https://github.com/gedcom/scCCESS" class="external-link">package</a>] [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Cao, Y., Lin, Y., Ormerod, J., Yang, P.,
Yang, J.Y.H. &amp; Lo, K.</strong> (2019) scDC: single cell differential
composition analysis. <em>BMC Bioinformatics</em>. [<a href="https://sydneybiox.github.io/scDC" class="external-link">package</a>] [<a href="https://doi.org/10.1186/s12859-019-3211-9" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Kim, T., Lo, K., Geddes, T., Kim, H.J.,
Yang, J.Y.H. &amp; Yang, P.</strong> (2019) scReClassify: post hoc cell
type classification of single-cell RNA-seq data. <em>BMC Genomics</em>.
[<a href="https://github.com/SydneyBioX/scReClassify" class="external-link">package</a>] [<a href="https://doi.org/10.1186/s12864-019-6305-x" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Lin, Y., Ghazanfar, S., Strbenac, D.,
Wang, A., Patrick, E.,</strong> Lin, D., Speed, T., <strong>Yang, J.Y.H.
&amp; Yang, P.</strong> (2019) Evaluating stably expressed genes in
single cells. <em>GigaScience</em>. [<a href="https://github.com/SydneyBioX/scMerge" class="external-link">package</a>] [<a href="https://doi.org/10.1093/gigascience/giz106" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> <strong>Baharlou, H., Canete, N. P.,</strong>
Cunningham, A. L., Harman, A. N., and <strong>Patrick, E.</strong>
(2019). Mass Cytometry Imaging for the Study of Human
Diseases—Applications and Data Analysis Strategies. <em>Frontiers in
Immunology</em>, 10.[<a href="https://doi.org/10.3389/fimmu.2019.02657" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
<div class="section level3">
<h3 id="section-5">2018<a class="anchor" aria-label="anchor" href="#section-5"></a>
</h3>
<ul>
<li><font size="+1"> <strong>Kim, T., Chen, I., Lin, Y., Wang, A., Yang,
J.Y.H. and Yang, P.</strong> (2018) Impact of similarity metrics on
single-cell RNA-seq data clustering. <em>Briefings in
Bioinformatics</em>. [<a href="https://taiyunkim.github.io/scClustBench" class="external-link">package</a>][<a href="https://doi.org/10.1093/bib/bby076" class="external-link">paper</a>]</font></li>
</ul>
</div>
<div class="section level3">
<h3 id="section-6">2016<a class="anchor" aria-label="anchor" href="#section-6"></a>
</h3>
<ul>
<li><font size="+1"> <strong>Ghazanfar, S.,</strong> Bisogni, A.,
<strong>Ormerod, J.T.,</strong>, Lin, D.M., and <strong>Yang,
J.Y.H.</strong> (2016) Integrated single cell data analysis reveals cell
specific networks and novel coactivation markers. <em>BMC Systems
Biology</em>. [<a href="https://doi.org/10.1186/s12918-016-0370-4" class="external-link">paper</a>]</font></li>
</ul>
</div>
</div>
<div class="section level2">
<h2 id="applications">
<strong>Applications</strong><a class="anchor" aria-label="anchor" href="#applications"></a>
</h2>
<ul>
<li><p><font size="+1"> Su, X., Zhao, L.N., Shi, Y., Zhang, R., Long,
Q., Bai, S., Luo, Q., <strong>Lin Y.,</strong>, Zou, X.,
<strong>Ghazanfar, S.,</strong>, Tao, Kun., Yang, G., Wang, L., He,
K.Y., Cui, X., He, J., Wu, J.X., Han, B., Wang, N., Li, X.,
<strong>Yang, P.</strong>, Hou, S., Sun, J., <strong>Yang,
J.Y.H.,</strong> Chen, J. and Han, Z.G. (2021) Single-cell and
single-variant resolution analysis of clonal evolution in human liver
cancer. <em>Biorxiv</em>. [<a href="https://doi.org/10.1101/2020.12.30.424907" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> Singh, R., Cottle, L., Loudovaris, T.,
<strong>Xiao, D., Yang. P.,</strong> Thomas, H., Kebede, A. M., and
Thorn, P. (2020) Enhanced structure and function of human pluripotent
stem cell-derived beta-cells cultured on extracellular matrix. <em>Stem
cells translational medicine</em>. [<a href="https://doi.org/10.1002/sctm.20-0224" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> Su, X., Long, Q., Bo, J., Shi, Y., Zhao, L.N.,
<strong>Lin Y.,</strong> Luo, Q., <strong>Ghazanfar, S.</strong>, Zhang,
C., Liu, Q., Wang, L., He, K., He, J., Cui, X., <strong>Yang,
J.Y.H.</strong>, Han, Z.G., Yang, G., and Sha, J.J. (2020) Mutational
and transcriptomic landscapes of a rare human prostate basal cell
carcinoma. <em>The Prostate</em>. [<a href="https://doi.org/10.1002/pros.23965" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> McGuire, H. M., Rizzetto, S., Withers, B. P.,
Clancy, L. E., Avdic, S., Stern, L., <strong>Patrick, E.</strong>,
Groth, B. F. d. S., Slobedman, B., Gottlieb, D. J., Luciani, F., and
Blyth, E. (2020). Mass cytometry reveals immune signatures associated
with cytomegalovirus (CMV) control in recipients of allogeneic
haemopoietic stem cell transplant and CMV-specific T cells. <em>Clinical
&amp; Translational Immunology</em>, 9(7):e1149.</font></p></li>
<li><p><font size="+1"> Jimenez Vera, E., Chew, Y. V., Nicholson, L.,
Burns, H., Anderson, P., Chen, H.-T., Williams, L., Keung, K., Zanjani,
N. T., Dervish, S., <strong>Patrick, E.</strong>, Wang, X. M., Yi, S.,
Hawthorne, W., Alexander, S., O’Connell, P. J., and Hu, M. (2019).
Standardisation of flow cytometry for whole blood immunopheno-typing of
islet transplant and transplant clinical trial recipients. <em>PLoS
ONE</em>, 14(5).</font></p></li>
<li><p><font size="+1"> He J., <strong>Lin Y.,</strong>, Su, X., Luo,
Q., <strong>Ghazanfar, S., Yang, J.Y.H.</strong> and Han,Z.G. (2019)
Single-Cell RNA-Seq Reveals Naïve B cells Associated with Better
Prognosis of HCC. <em>Biorxiv</em>. [<a href="https://doi.org/10.1101/731935" class="external-link">paper</a>]</font></p></li>
<li><p><font size="+1"> Su, X., Shi, Y., Zou, X., Lu, Z.N., Xie, G.,
<strong>Yang, J.Y.H.,</strong> Wu, C.C., Cui., X.F., He, K.Y., Luo, Q.,
Qu, Y.L., Wang, N., Wang, L., Han, Z.G. (2017) Single-cell RNA-Seq
analysis reveals dynamic trajectories during mouse liver development
<em>BMC Genomics</em>. [<a href="https://doi.org/10.1186/s12864-017-4342-x" class="external-link">paper</a>]</font></p></li>
</ul>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Sydney Precision Bioinformatics Alliance.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
