<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="A collection of single cell RNA-sequencing data analysis functions including data integration, clustering, cell type annotation and cell identity characterisation.">
<title>Single cell data integrative analysis • scdney</title>
<script src="deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="pkgdown.js"></script><meta property="og:title" content="Single cell data integrative analysis">
<meta property="og:description" content="A collection of single cell RNA-sequencing data analysis functions including data integration, clustering, cell type annotation and cell identity characterisation.">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-dark"><div class="container">
    
    <a class="navbar-brand me-2" href="index.html">scdney</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.2.4</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="articles/website/package.html">
    <span class="fa fa-laptop-code fa-lg"></span>
     
    Packages
  </a>
</li>
<li class="nav-item">
  <a class="nav-link" href="articles/website/publication.html">
    <span class="fa fa-book fa-lg"></span>
     
    Publications
  </a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/SydneyBioX/scdney">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-home">
<div class="row">
  <main id="main" class="col-md-9"><div class="section level1">
<div class="page-header"><h1 id="scdney---single-cell-data-integrative-analysis">scdney - Single cell data integrative analysis<a class="anchor" aria-label="anchor" href="#scdney---single-cell-data-integrative-analysis"></a>
</h1></div>
<!--
<br />
-->
<!--
[![](https://img.shields.io/github/last-commit/SydneyBioX/scdney.svg)](https://github.com/SydneyBioX/scdney/commits/master) [![Travis build status](https://travis-ci.org/SydneyBioX/scdney.svg?branch=master)](https://travis-ci.org/SydneyBioX/scdney)


<br />

-->
<p><font size="+1"> <code>scdney</code> is a wrapper package with collection of single cell analysis R packages developed by team of <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics" class="external-link">Sydney Precision Bioinformatics Alliance</a> at The University of Sydney. </font></p>
<p><font size="+1"> More information about the <code>scdney</code> package: <a href="https://sydneybiox.github.io/scdney/" class="external-link uri">https://sydneybiox.github.io/scdney/</a>. </font></p>
<p><img src="reference/figures/scdney_packages.png" align="right" width="300" hspace="20" vspace="10"></p>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>Before running the installation command below, some <code>R</code> dependencies must be installed. This can be done as follows:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"BiocManager"</span>,<span class="st">"remotes"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p>To install all packages in available <code>scdney</code> and attached the installed <code>scdney</code> packages in the current R session.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># install scdney packages</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"SydneyBioX/scdney"</span><span class="op">)</span></span>
<span><span class="co"># load  scdney packages</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">scdney</span><span class="op">)</span></span></code></pre></div>
<div class="section level3">
<h3 id="installation-notes">Installation Notes<a class="anchor" aria-label="anchor" href="#installation-notes"></a>
</h3>
<div class="section level4">
<h4 id="installation-time">Installation Time<a class="anchor" aria-label="anchor" href="#installation-time"></a>
</h4>
<p>Installing the whole suite of packages can take a long time on UNIX-based systems:</p>
<ul>
<li>Around 6 mins when using prebuilt binaries from <a href="https://eddelbuettel.github.io/r2u/" class="external-link">r2u</a> (discussed below). Unfortunately, this only works on Ubuntu LTS systems.</li>
<li>Around 20 mins on a basic r installation including some basic Bioconductor packages (e.g., <a href="https://github.com/bioconductor/bioconductor_docker" class="external-link">bioconductor_docker</a> image).</li>
<li>More than 1 hour on system with a basic r installation (e.g., <a href="https://hub.docker.com/_/r-base/" class="external-link">r-base</a> image)</li>
</ul>
<p>If compiling the binaries locally is a priority, this issue can be partially addressed by adding <code>MAKE='make -j NCORES'</code> to one’s <code>Renviron</code> file.</p>
<ul>
<li>
<code>NCORES</code> should be replaced be the number of jobs to run in parallel.</li>
</ul>
<p>Alternatively, this process can be made significantly faster by using <a href="https://eddelbuettel.github.io/r2u/" class="external-link"><code>r2u</code></a>. This project builds <code>r</code> packages for Ubuntu LTS repository system and integrates them with the system package manager. This will lead to significant increases in performance. An example docker build image for this method is included in the <code>/build</code> directory.</p>
</div>
<div class="section level4">
<h4 id="non-r-dependencies">Non-R Dependencies<a class="anchor" aria-label="anchor" href="#non-r-dependencies"></a>
</h4>
<p>There are several non-R dependencies that must be installed in order for <code>scdney</code> to install correctly. Most of these are installed by default, but if there is a problem in the installation, one of this system requirements is likely to be the culprit.</p>
<ul>
<li>
<code>make</code>, <code>libpng-dev</code>, <code>pandoc</code>, <code>libjpeg-dev</code>, <code>zlib1g-dev</code>, <code>libfreetype6-dev</code>, <code>libfribidi-dev</code>, <code>libharfbuzz-dev</code>, <code>libxml2-dev</code>, <code>libfontconfig1-dev</code>, <code>pandoc-citeproc</code>, <code>cmake</code>, <code>libicu-dev</code>, <code>libssl-dev</code>, <code>libglpk-dev</code>, <code>libgmp3-dev</code>, <code>libtiff-dev</code>, <code>libcurl4-openssl-dev</code>
</li>
</ul>
</div>
</div>
</div>
<div class="section level2">
<h2 id="scdney-workshops-series">
<code>scdney</code> Workshops Series<a class="anchor" aria-label="anchor" href="#scdney-workshops-series"></a>
</h2>
<ul>
<li>
<a href="https://sydneybiox.github.io/BIS2019_SC/" class="external-link"><strong>Single-cell analysis fundation workshop</strong></a>, where we use two mouse liver datasets to illustrate three critical topics in scRNA-seq analysis: Quality control of scRNA-seq data, data integration of multiple scRNA-seq data, and cell type marker identification.</li>
<li>
<a href="https://sydneybiox.github.io/scdneyAdvancedPhenotyping/articles/advanced_phenotyping.html" class="external-link"><strong>Advanced Phenotyping using scdney</strong></a>, where we describe various strategies for cellular phenotyping, construct and characterise trajectory, and identify cell-cell interaction.</li>
<li>
<a href="https://sydneybiox.github.io/BiocAsia2020CiteFuse/articles/CiteFuse_BioCAsia_workshop.html" class="external-link"><strong>CITE-seq data analysis using CiteFuse</strong></a>, a hands-on experience to the CiteFuse package workshop.</li>
</ul>
</div>
</div>
  </main><aside class="col-md-3"><div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li>What license is it under?</li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing scdney</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Sydney Precision Bioinformatics Alliance <br><small class="roles"> Maintainer </small>  </li>
</ul>
</div>



  </aside>
</div>


    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Sydney Precision Bioinformatics Alliance.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
