<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>scClassify â€¢ scdney</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="scClassify">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scdney</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.5</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/scdney.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/scClassify.html">scClassify</a>
    </li>
    <li>
      <a href="../articles/scDC.html">scDC: single cell differential composition analysis</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>scClassify</h1>
                        <h4 class="author">Yingxin Lin</h4>
            
            <h4 class="date">2019-08-22</h4>
      
      
      <div class="hidden name"><code>scClassify.Rmd</code></div>

    </div>

    
    
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(scdney)</code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/data">data</a></span>(<span class="st">"scClassify_example"</span>)
xin_cellTypes &lt;-<span class="st"> </span>scClassify_example<span class="op">$</span>xin_cellTypes
exprsMat_xin_subset &lt;-<span class="st"> </span>scClassify_example<span class="op">$</span>exprsMat_xin_subset
wang_cellTypes &lt;-<span class="st"> </span>scClassify_example<span class="op">$</span>wang_cellTypes
exprsMat_wang_subset &lt;-<span class="st"> </span>scClassify_example<span class="op">$</span>exprsMat_wang_subset</code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/cat">cat</a></span>(<span class="st">"Cell type composition of Xin et al. data"</span>)
<span class="co">#&gt; Cell type composition of Xin et al. data</span>
<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/table">table</a></span>(xin_cellTypes)
<span class="co">#&gt; xin_cellTypes</span>
<span class="co">#&gt; alpha  beta delta gamma </span>
<span class="co">#&gt;   285   261    49    79</span>
<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/cat">cat</a></span>(<span class="st">"Cell type composition of Wang et al. data"</span>)
<span class="co">#&gt; Cell type composition of Wang et al. data</span>
<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/table">table</a></span>(wang_cellTypes)
<span class="co">#&gt; wang_cellTypes</span>
<span class="co">#&gt;   acinar    alpha     beta    delta   ductal    gamma stellate </span>
<span class="co">#&gt;        5      206      118       10       96       21       45</span></code></pre></div>
<div id="scclassify" class="section level1">
<h1 class="hasAnchor">
<a href="#scclassify" class="anchor"></a>scClassify</h1>
<div id="non-ensemble-scclassify" class="section level2">
<h2 class="hasAnchor">
<a href="#non-ensemble-scclassify" class="anchor"></a>Non-ensemble scClassify</h2>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">scClassify_res &lt;-<span class="st"> </span><span class="kw"><a href="../reference/scClassify.html">scClassify</a></span>(<span class="dt">exprsMat_train =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/matrix">as.matrix</a></span>(exprsMat_xin_subset),
                      <span class="dt">cellTypes_train =</span> xin_cellTypes,
                      <span class="dt">exprsMat_test =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="dt">wang =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/matrix">as.matrix</a></span>(exprsMat_wang_subset)),
                      <span class="dt">cellTypes_test =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="dt">wang =</span> wang_cellTypes),
                      <span class="dt">tree =</span> <span class="st">"HOPACH"</span>,
                      <span class="dt">algorithm =</span> <span class="st">"WKNN"</span>,
                      <span class="dt">selectFeatures =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"limma"</span>),
                      <span class="dt">similarity =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"pearson"</span>),
                      <span class="dt">weighted_ensemble =</span> <span class="ot">TRUE</span>,
                      <span class="dt">verbose =</span> <span class="ot">FALSE</span>)
<span class="co">#&gt; after filtering not expressed genes </span>
<span class="co">#&gt; [1] 980 674</span></code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/table">table</a></span>(scClassify_res<span class="op">$</span>testRes<span class="op">$</span>wang<span class="op">$</span>pearson_WKNN_limma<span class="op">$</span>predRes, wang_cellTypes)
<span class="co">#&gt;                   wang_cellTypes</span>
<span class="co">#&gt;                    acinar alpha beta delta ductal gamma stellate</span>
<span class="co">#&gt;   alpha                 0   206    0     0      0     2        0</span>
<span class="co">#&gt;   beta                  0     0  118     0      1     0        0</span>
<span class="co">#&gt;   beta_delta_gamma      0     0    0     0     20     0        0</span>
<span class="co">#&gt;   delta                 0     0    0    10      0     0        0</span>
<span class="co">#&gt;   gamma                 0     0    0     0      0    19        0</span>
<span class="co">#&gt;   unassigned            5     0    0     0     75     0       45</span></code></pre></div>
</div>
<div id="ensemble-scclassify" class="section level2">
<h2 class="hasAnchor">
<a href="#ensemble-scclassify" class="anchor"></a>Ensemble scClassify</h2>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">scClassify_res_ensemble &lt;-<span class="st"> </span><span class="kw"><a href="../reference/scClassify.html">scClassify</a></span>(<span class="dt">exprsMat_train =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/matrix">as.matrix</a></span>(exprsMat_xin_subset),
                      <span class="dt">cellTypes_train =</span> xin_cellTypes,
                      <span class="dt">exprsMat_test =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="dt">wang =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/matrix">as.matrix</a></span>(exprsMat_wang_subset)),
                      <span class="dt">cellTypes_test =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="dt">wang =</span> wang_cellTypes),
                      <span class="dt">tree =</span> <span class="st">"HOPACH"</span>,
                      <span class="dt">algorithm =</span> <span class="st">"WKNN"</span>,
                      <span class="dt">selectFeatures =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"limma"</span>),
                      <span class="dt">similarity =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"pearson"</span>, <span class="st">"spearman"</span>, <span class="st">"cosine"</span>),
                      <span class="dt">weighted_ensemble =</span> <span class="ot">TRUE</span>,
                      <span class="dt">verbose =</span> <span class="ot">FALSE</span>)
<span class="co">#&gt; after filtering not expressed genes </span>
<span class="co">#&gt; [1] 980 674</span></code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/table">table</a></span>(scClassify_res_ensemble<span class="op">$</span>ensembleRes<span class="op">$</span>wang<span class="op">$</span>cellTypes, wang_cellTypes)
<span class="co">#&gt;                   wang_cellTypes</span>
<span class="co">#&gt;                    acinar alpha beta delta ductal gamma stellate</span>
<span class="co">#&gt;   alpha                 0   206    0     0      0     2        1</span>
<span class="co">#&gt;   unassigned            5     0    0     0     81     0       44</span>
<span class="co">#&gt;   delta                 0     0    0    10      0     0        0</span>
<span class="co">#&gt;   beta_delta_gamma      0     0    0     0     13     0        0</span>
<span class="co">#&gt;   beta                  0     0  118     0      2     0        0</span>
<span class="co">#&gt;   gamma                 0     0    0     0      0    19        0</span></code></pre></div>
</div>
</div>
<div id="sample-size-calculation" class="section level1">
<h1 class="hasAnchor">
<a href="#sample-size-calculation" class="anchor"></a>Sample Size Calculation</h1>
<p>For the purpose of illustration, here we similate an accuracy matrix</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/Random">set.seed</a></span>(<span class="dv">2019</span>)
n &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/seq">seq</a></span>(<span class="dv">20</span>, <span class="dv">1000</span>, <span class="dv">10</span>), <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/seq">seq</a></span>(<span class="dv">1100</span>, <span class="dv">10000</span>, <span class="dv">100</span>))

accMat &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/do.call">do.call</a></span>(cbind, <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">lapply</a></span>(<span class="dv">1</span><span class="op">:</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(n), <span class="cf">function</span>(i){
  tmp_n &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/rep">rep</a></span>(n[i], <span class="dv">50</span>)
  y &lt;-<span class="st"> </span><span class="op">-</span><span class="dv">2</span><span class="op">/</span>(tmp_n<span class="op">^</span><span class="fl">0.8</span>) <span class="op">+</span><span class="st"> </span><span class="fl">0.95</span> <span class="op">+</span><span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/stats/topics/Normal">rnorm</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(tmp_n), <span class="dv">0</span>, <span class="fl">0.02</span>)
  y
}))</code></pre></div>
<div id="fitting-learning-curve" class="section level2">
<h2 class="hasAnchor">
<a href="#fitting-learning-curve" class="anchor"></a>Fitting Learning Curve</h2>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">res &lt;-<span class="st"> </span><span class="kw"><a href="../reference/learningCurve.html">learningCurve</a></span>(<span class="dt">accMat =</span> accMat, n, <span class="dt">verbose =</span> <span class="ot">FALSE</span>)
res<span class="op">$</span>model<span class="op">$</span>mean
<span class="co">#&gt; Nonlinear regression model</span>
<span class="co">#&gt;   model: acc ~ I(1/n^(c) * a) + b</span>
<span class="co">#&gt;    data: dat_train</span>
<span class="co">#&gt;                 a.(Intercept) c.log(max(acc) + 0.001 - acc) </span>
<span class="co">#&gt;                       -2.0426                        0.8029 </span>
<span class="co">#&gt;                             b </span>
<span class="co">#&gt;                        0.9493 </span>
<span class="co">#&gt;  residual sum-of-squares: 0.001714</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Number of iterations to convergence: 12 </span>
<span class="co">#&gt; Achieved convergence tolerance: 1.49e-08</span>
res<span class="op">$</span>plot</code></pre></div>
<p><img src="scClassify_files/figure-html/unnamed-chunk-9-1.png" width="768"></p>
<p>To get the sample size for future experiment given the accuracy rate</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/getN.html">getN</a></span>(res, <span class="dt">acc =</span> <span class="fl">0.93</span>)
<span class="co">#&gt; [1] 331</span></code></pre></div>
</div>
</div>
<div id="session-info" class="section level1">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session Info</h1>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/sessionInfo">sessionInfo</a></span>()
<span class="co">#&gt; R version 3.6.1 (2019-07-05)</span>
<span class="co">#&gt; Platform: x86_64-apple-darwin15.6.0 (64-bit)</span>
<span class="co">#&gt; Running under: macOS Mojave 10.14.6</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Matrix products: default</span>
<span class="co">#&gt; BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib</span>
<span class="co">#&gt; LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; locale:</span>
<span class="co">#&gt; [1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; attached base packages:</span>
<span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; other attached packages:</span>
<span class="co">#&gt; [1] scdney_0.1.4</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; loaded via a namespace (and not attached):</span>
<span class="co">#&gt;   [1] snow_0.4-3                  backports_1.1.4            </span>
<span class="co">#&gt;   [3] Hmisc_4.2-0                 blme_1.0-4                 </span>
<span class="co">#&gt;   [5] plyr_1.8.4                  igraph_1.2.4.1             </span>
<span class="co">#&gt;   [7] lazyeval_0.2.2              splines_3.6.1              </span>
<span class="co">#&gt;   [9] BiocParallel_1.18.0         GenomeInfoDb_1.20.0        </span>
<span class="co">#&gt;  [11] ggplot2_3.2.0               amap_0.8-17                </span>
<span class="co">#&gt;  [13] digest_0.6.20               foreach_1.4.4              </span>
<span class="co">#&gt;  [15] htmltools_0.3.6             viridis_0.5.1              </span>
<span class="co">#&gt;  [17] magrittr_1.5                checkmate_1.9.4            </span>
<span class="co">#&gt;  [19] memoise_1.1.0               cluster_2.1.0              </span>
<span class="co">#&gt;  [21] doParallel_1.0.14           mixtools_1.1.0             </span>
<span class="co">#&gt;  [23] limma_3.40.2                recipes_0.1.6              </span>
<span class="co">#&gt;  [25] gower_0.2.1                 matrixStats_0.54.0         </span>
<span class="co">#&gt;  [27] pkgdown_1.3.0               colorspace_1.4-1           </span>
<span class="co">#&gt;  [29] ggrepel_0.8.1               pan_1.6                    </span>
<span class="co">#&gt;  [31] xfun_0.8                    dplyr_0.8.3                </span>
<span class="co">#&gt;  [33] crayon_1.3.4                RCurl_1.95-4.12            </span>
<span class="co">#&gt;  [35] roxygen2_6.1.1              lme4_1.1-21                </span>
<span class="co">#&gt;  [37] survival_2.44-1.1           iterators_1.0.10           </span>
<span class="co">#&gt;  [39] glue_1.3.1                  polyclip_1.10-0            </span>
<span class="co">#&gt;  [41] gtable_0.3.0                ipred_0.9-9                </span>
<span class="co">#&gt;  [43] zlibbioc_1.30.0             XVector_0.24.0             </span>
<span class="co">#&gt;  [45] DelayedArray_0.10.0         SingleCellExperiment_1.6.0 </span>
<span class="co">#&gt;  [47] BiocGenerics_0.30.0         jomo_2.6-8                 </span>
<span class="co">#&gt;  [49] abind_1.4-5                 scales_1.0.0               </span>
<span class="co">#&gt;  [51] mvtnorm_1.0-11              Rcpp_1.0.1                 </span>
<span class="co">#&gt;  [53] viridisLite_0.3.0           htmlTable_1.13.1           </span>
<span class="co">#&gt;  [55] proxy_0.4-23                mclust_5.4.5               </span>
<span class="co">#&gt;  [57] foreign_0.8-71              Formula_1.2-3              </span>
<span class="co">#&gt;  [59] stats4_3.6.1                lava_1.6.5                 </span>
<span class="co">#&gt;  [61] prodlim_2018.04.18          htmlwidgets_1.3            </span>
<span class="co">#&gt;  [63] RColorBrewer_1.1-2          acepack_1.4.1              </span>
<span class="co">#&gt;  [65] mice_3.6.0                  pkgconfig_2.0.2            </span>
<span class="co">#&gt;  [67] manipulate_1.0.1            farver_1.1.0               </span>
<span class="co">#&gt;  [69] nnet_7.3-12                 caret_6.0-84               </span>
<span class="co">#&gt;  [71] labeling_0.3                tidyselect_0.2.5           </span>
<span class="co">#&gt;  [73] rlang_0.4.0                 reshape2_1.4.3             </span>
<span class="co">#&gt;  [75] pbmcapply_1.5.0             munsell_0.5.0              </span>
<span class="co">#&gt;  [77] tools_3.6.1                 generics_0.0.2             </span>
<span class="co">#&gt;  [79] broom_0.5.2                 ggridges_0.5.1             </span>
<span class="co">#&gt;  [81] evaluate_0.14               stringr_1.4.0              </span>
<span class="co">#&gt;  [83] yaml_2.2.0                  ModelMetrics_1.2.2         </span>
<span class="co">#&gt;  [85] knitr_1.23                  fs_1.3.1                   </span>
<span class="co">#&gt;  [87] randomForest_4.6-14         purrr_0.3.2                </span>
<span class="co">#&gt;  [89] mitml_0.3-7                 dendextend_1.12.0          </span>
<span class="co">#&gt;  [91] ggraph_1.0.2                nlme_3.1-140               </span>
<span class="co">#&gt;  [93] xml2_1.2.0                  compiler_3.6.1             </span>
<span class="co">#&gt;  [95] rstudioapi_0.10             e1071_1.7-2                </span>
<span class="co">#&gt;  [97] statmod_1.4.32              tibble_2.1.3               </span>
<span class="co">#&gt;  [99] tweenr_1.0.1                DescTools_0.99.28          </span>
<span class="co">#&gt; [101] stringi_1.4.3               desc_1.2.0                 </span>
<span class="co">#&gt; [103] hopach_2.44.0               lattice_0.20-38            </span>
<span class="co">#&gt; [105] Matrix_1.2-17               commonmark_1.7             </span>
<span class="co">#&gt; [107] nloptr_1.2.1                pillar_1.4.2               </span>
<span class="co">#&gt; [109] data.table_1.12.2           bitops_1.0-6               </span>
<span class="co">#&gt; [111] GenomicRanges_1.36.0        R6_2.4.0                   </span>
<span class="co">#&gt; [113] latticeExtra_0.6-28         gridExtra_2.3              </span>
<span class="co">#&gt; [115] IRanges_2.18.1              codetools_0.2-16           </span>
<span class="co">#&gt; [117] boot_1.3-22                 MASS_7.3-51.4              </span>
<span class="co">#&gt; [119] assertthat_0.2.1            SummarizedExperiment_1.14.0</span>
<span class="co">#&gt; [121] MAST_1.10.0                 rprojroot_1.3-2            </span>
<span class="co">#&gt; [123] minpack.lm_1.2-1            withr_2.1.2                </span>
<span class="co">#&gt; [125] S4Vectors_0.22.0            GenomeInfoDbData_1.2.1     </span>
<span class="co">#&gt; [127] diptest_0.75-7              expm_0.999-4               </span>
<span class="co">#&gt; [129] parallel_3.6.1              doSNOW_1.0.16              </span>
<span class="co">#&gt; [131] grid_3.6.1                  rpart_4.1-15               </span>
<span class="co">#&gt; [133] timeDate_3043.102           tidyr_0.8.3                </span>
<span class="co">#&gt; [135] class_7.3-15                minqa_1.2.4                </span>
<span class="co">#&gt; [137] rmarkdown_1.14              segmented_1.0-0            </span>
<span class="co">#&gt; [139] clusteval_0.1               ggforce_0.2.2              </span>
<span class="co">#&gt; [141] Biobase_2.44.0              lubridate_1.7.4            </span>
<span class="co">#&gt; [143] base64enc_0.1-3</span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li>
<a href="#scclassify">scClassify</a><ul class="nav nav-pills nav-stacked">
<li><a href="#non-ensemble-scclassify">Non-ensemble scClassify</a></li>
      <li><a href="#ensemble-scclassify">Ensemble scClassify</a></li>
      </ul>
</li>
      <li>
<a href="#sample-size-calculation">Sample Size Calculation</a><ul class="nav nav-pills nav-stacked">
<li><a href="#fitting-learning-curve">Fitting Learning Curve</a></li>
      </ul>
</li>
      <li><a href="#session-info">Session Info</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by Taiyun Kim.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
